python_modules/dagster/dagster/_core/definitions/asset_selection.py

```diff
@@ -969,16 +1029,17 @@ def resolve_inner(
         return {key for key in base_set if asset_graph.get(key).tags.get(self.key) == self.value}
 
     def to_selection_str(self) -> str:
-        if self.value:
-            return f'tag:"{self.key}"="{self.value}"'
-        else:
+        if self.key is None:
+            return "tag:<null>"
```

Comment:
this is the bit that I'm not sure exactly what it'd even mean -- does it just mean that this has no tags at all? seems a bit niche and not super reliable as we automatically add tags for a variety of different reasons (e.g. all external assets actually get a tag), so i'd rather only support checking if values were null